modules/openedx/application/data.tf

```diff
@@ -4,7 +4,7 @@ data "aws_subnet_ids" "subnets" {
 
 data "aws_acm_certificate" "certificate" {
   domain      = var.aws_acm_certificate_domain
-  statuses    = ["ISSUED"]
+  statuses    = ["ISSUED", "PENDING_VALIDATION"]
```

Comment:
@pomegranited this is a module that we are not using in our case, so I wouldn't know if we could potentially affect those who are using this one. Maybe nothing to worry about.

@giovannicimolin care to comment here?

Reply:
There's a big issue with this repo currently: we have to terraform approaches that do the same thing (provision Open edX environment).

The `openedx` module is being used by `KKUx`, `NELP` and `Baobab` stage so far. At first there was only this deployment method, which was supposed to be our main and unified approach.

I don't think this will break any of the current instances. In fact, it's going to unblock deployments where we don't have the DNS records set up yet, so :+1: .

Reply:
Btw, I've created https://github.com/open-craft/terraform-scripts/issues/9 to try to address this.

Reply:
Also: @pomegranited I've run a `terraform plan` with this branch on NELP's setup and the plan came out clean, so no issues  with the changes here.

Reply:
Awesome, thank you for checking that this doesn't break NELP, @giovannicimolin !

I didn't realise that we've got two mechanisms in the same repo.. that will get ugly fast. Could you create a ticket for next sprint to address this?

Reply:
@pomegranited @eLRuLL https://tasks.opencraft.com/browse/BB-3586

I'll probably need some help from you guys since you are already using the other scripts (merging the approaches might require reprovisioning a few things and renaming others).

Reply:
Noted, thank you @giovannicimolin !